app/scripts/services/error.js

```diff
@@ -6,6 +6,8 @@ angular.module('confRegistrationWebApp')
       // Generate a promise catch handler that generates an Error object from an HTTP response object
       errorFromResponse: function (defaultErrorMessage) {
         return function (res) {
+          console.error(res);
```

Comment:
✔️  Yes, that logging is intentional, but I just switched it to use `$log` since that is best-practice for Angular apps.